packages/zosmf/src/CheckStatus.ts

```diff
@@ -36,6 +36,10 @@ export class CheckStatus {
         return ZosmfRestClient.getExpectJSON(session, infoEndpoint);
     }
 
+    public static async isZosVersionGreaterThan(session: AbstractSession, version: string): Promise<boolean> {
```

Comment:
From the implementation, should this function be named IsZosVersionAtLeast instead of GreaterThan?